node-playwright-firefox/Dockerfile

```diff
@@ -23,7 +18,54 @@ RUN groupadd -r myuser && useradd -r -g myuser -G audio,video myuser \
 RUN npm config --global set update-notifier false
 
 # Install all required playwright dependencies for firefox
-RUN PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1 npm_config_ignore_scripts=1 npx playwright install-deps firefox
+# Keep updated from https://github.com/microsoft/playwright/blob/a06b06b82bb6b2d550f12e8b18af298f23a08828/packages/playwright-core/src/server/registry/nativeDeps.ts#L229
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
```

Comment:
So we had the deps. Then we switched to `npx ...` and now we're switching back to deps? Might be worth making a comment about what's going on here 😄

Reply:
For some reason it stopped working and now returns a `Cannot install dependencies for this linux distribution!` (which makes sense since it expects Ubuntu, and not debian...but this used to work too..)
I'll see if I can figure out whats going on and submit a PR to go *back* to the CLI if it works (but at this point it's probably easier to manually update the list from their file